tools/scripts/upload-ab-to-s3.js

```diff
@@ -205,3 +206,8 @@ function getOpenPrNums () {
     })
   })
 }
+
+async function getExperiments () {
+  const objects = await getListOfObjects(bucket, 'experiments/')
+  return objects.filter(({ Key }) => Key.includes('nr-loader-spa.min.js')).map(({ Key }) => Key)
```

Comment:
So are we just going with a flat folder for experiments for now? Not supporting multiple experiments?

Reply:
it will be a single nesting under `experiments/`

So basically `experiments/<branch-name>/nr-loader-spa.min.js` for each branch name that triggers an action.


The response from "getExperiments" should be a list of full truncated paths
```javascript
[
    "experiments/initial-test/nr-loader-spa.min.js",
    "experiments/rrweb/nr-loader-spa.min.js",
    // etc...
 ]
 ```